src/components/LogIn/LogIn.tsx

```diff
@@ -1,17 +1,15 @@
 import {  Button, Group, TextInput, PasswordInput} from '@mantine/core';
 import { useForm } from '@mantine/form';
 import './LogIn.css'
+import { initialValues } from './constants';
+import { validateEmail,validatePassword } from './validation';
 export function Login(){
     const form=useForm({
         mode:'uncontrolled',
-        initialValues:{
-            email:'',
-            parola:'',
-            termeniSiConditii:'',
-        },
+        initialValues :initialValues,
         validate: {
```

Comment:
de ce nu punem tot obiectul intr-o constanta si doar il pasam:
validate: loginValidation

si loginValidation este obiectul in sine 🤯

Reply:
ups...incerc sa corectez acum. Initial ma gandeam ca doar pentru initialValues